docs/en/basic/Command-Line-Interface.md

```diff
@@ -132,105 +131,104 @@ $ ./bin/alluxio getConf --unit S alluxio.master.journal.flush.timeout
+where:
+* `--logName <arg>` indicates the logger's class (e.g. `alluxio.master.file.DefaultFileSystemMaster`)
+* `--target <arg>` lists the Alluxio master or workers to set.
+The target could be of the form `<master|workers|host:webPort>` and multiple targets can be listed as comma-separated entries.
+`host:webPort` pair must be one of the workers. TODO: what does this mean??? must be???
+The default target value is all masters and workers.
+* `--level <arg>` If provided, the command changes to the given logger level, 
+otherwise it returns the current logger level.
 
-For example, this command sets the class `alluxio.heartbeat.HeartbeatContext`'s logger level to
-DEBUG on master as well as a worker at `192.168.100.100:30000`.
+For example, the following command sets the logger level of the class `alluxio.heartbeat.HeartbeatContext` to
+`DEBUG` on master as well as a worker at `192.168.100.100:30000`:
 
 ```bash
 alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=master,192.168.100.100:30000 --level=DEBUG
 ```
 
-And the following command gets all workers' log level on class `alluxio.heartbeat.HeartbeatContext`
+And the following command returns the log level of the class `alluxio.heartbeat.HeartbeatContext` among all the workers:
 ```bash
 alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=workers
 ```
 
 ### runTests
 
-The `runTests` command will run all end-to-end tests on an Alluxio cluster 
-and provide a comprehensive sanity check.
+The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a comprehensive sanity check.
 
 ```bash
 $ ./bin/alluxio runTests
 ```
 
 ### upgradeJournal
 
-The `upgradeJournal` command will upgrade an Alluxio journal version 0 (Alluxio version < 1.5.0) 
+The `upgradeJournal` command upgrades an Alluxio journal version 0 (Alluxio version < 1.5.0) 
 to an Alluxio journal version 1 (Alluxio version >= 1.5.0).
 
-`-journalDirectoryV0 <arg>` will provide the v0 journal persisted location. It is assumed to be 
-the same as the v1 journal directory if not set.
+`-journalDirectoryV0 <arg>` will provide the v0 journal persisted location.\
+It is assumed to be the same as the v1 journal directory if not set.
 
 ```bash
 $ ./bin/alluxio upgradeJournal
 ```
 
 ### copyDir
 
-The `copyDir` command will copy the `PATH` to all worker nodes listed in `conf/workers`.
+The `copyDir` command copies the directory at `PATH` to all worker nodes listed in `conf/workers`.
 
 ```bash
 $ ./bin/alluxio copyDir conf/alluxio-site.properties
 ```
 
 ### version
 
-The `version` command will print Alluxio version and exit.
+The `version` command prints Alluxio version.
 
 ```bash
 $ ./bin/alluxio version
 ```
 
 ### validateConf
 
-The `validateConf` command will validate Alluxio conf and exit.
+The `validateConf` command validates the Alluxio configuration. TODO: does this validates it across every master and worker???
```

Comment:
-The `validateConf` command validates the Alluxio configuration. TODO: does this validates it across every master and worker???
+The `validateConf` command validates the local Alluxio configuration files, checking for common misconfigurations.